SmartApi/smartConnect.py

```diff
@@ -433,6 +436,30 @@ def searchScrip(self, exchange, searchscrip):
         else:
             return searchScripResult
         
+    def make_authenticated_get_request(self, url, access_token):
+        headers = self.requestHeaders()
+        if access_token:
+            headers["Authorization"] = "Bearer " + access_token
+        response = requests.get(url, headers=headers)
+        if response.status_code == 200:
+            data = json.loads(response.text)
+            return data
+        else:
+            raise Exception(f"Error: {response.status_code}, {response.text}")
+        
+    def ind_order_details(self, qParam):
```

Comment:
make method name - individual_order_detail

Reply:
rename the function name ind_order_detail to individual_order_detail